db/migrate/20151124183140_remove_talks.rb

```diff
@@ -1,5 +1,6 @@
 class RemoveTalks < ActiveRecord::Migration
   def change
-    drop_table :talks
+    drop_table :talks do
```

Comment:
It does make sense, But i guess that instead of a having a method's behavior hijacked because of the context where it is executed, I would rather have used `up` and `down` in place of `change` like in the old days!
But, as I said, I am just grumpy :older_man:

It wouldn't happen with Phoenix :trollface: